lib/rucio/web/rest/webpy/v1/identity.py

```diff
@@ -63,18 +63,20 @@ def PUT(self, account):
         """
         username = ctx.env.get('HTTP_X_RUCIO_USERNAME')
         password = ctx.env.get('HTTP_X_RUCIO_PASSWORD')
+        email = ''
```

Comment:
Just added a blanc mail because the test I added to test_identity.py failed on postgres and oracle because of a DB constraint that email should not be none. Can also fix it in this PR properly or open a new issue.